From e98ba27f95a2c137fe836bf342de427916df9f5a Mon Sep 17 00:00:00 2001
From: Simon McVittie <simon.mcvittie@collabora.co.uk>
Date: Wed, 18 Dec 2013 20:01:01 +0000
Subject: BSConfig.pm.template: include a skeleton reprepro setup

Bug: Collabora #397
Reviewed-by: Sjoerd Simons <sjoerd.simons@collabora.co.uk>

--- a/src/backend/BSConfig.pm.template
+++ b/src/backend/BSConfig.pm.template
@@ -202,6 +202,25 @@
 #                           'main' => 'http://main-backend-server:5252',
 #                         };
 
+# Manage these projects' repositories with reprepro instead of using
+# dpkg-scanpackages/dpkg-scansources. Before enabling this, you will
+# need to create /srv/obs/repos/shared/exampledistro/
+# with a paragraph in its conf/distributions file containing
+# "Codename: examplesuite" and "Components: examplecomponent othercomponent"
+#
+#our $reprepository = {
+#  "exampledistro:examplesuite:examplecomponent/debian_wheezy_main_main" => {
+#    "repository" => "shared/exampledistro",
+#    "codename" => "examplesuite",
+#    "component" => "examplecomponent",
+#  },
+#  "exampledistro:examplesuite:othercomponent/debian_wheezy_main_main" => {
+#    "repository" => "shared/exampledistro",
+#    "codename" => "examplesuite",
+#    "component" => "othercomponent",
+#  },
+#};
+
 # host specific configs
 my $hostconfig = __FILE__;
 $hostconfig =~ s/[^\/]*$/bsconfig.$hostname/;
