Description: drop test and development depends.
Author: Andrew Lee (李健秋) <ajqlee@debian.org>

---
--- open-build-service-2.6.0.orig/src/api/Gemfile
+++ open-build-service-2.6.0/src/api/Gemfile
@@ -53,50 +53,6 @@ group :production do
 #  gem 'newrelic_rpm'
 end
 
-# Gems used only for testing the application and not required in production environments by default.
-group :test do
-  # as testing frameworks
-  # to update to 5.3.4, we need to fix randomization
-  # see https://github.com/seattlerb/minitest/issues/431
-##  gem 'minitest', '= 5.3.3'
-  gem 'minitest', '>= 5.3.3'
-  # to ensure a clean state for testing
-  gem 'database_cleaner', '>= 1.0.1'
-  # for jenkins
-  gem 'ci_reporter'
-  # for test coverage reports
-  gem 'simplecov', require: false
-  # to colorize minitest output
-  # gem 'minitest-colorize'
-  # for rspec like matchers
-  gem 'capybara_minitest_spec'
-  # to generate random long strings
-  gem 'faker'
-  # to freeze time
-  gem 'timecop'
-  # to fake backend replies
-  gem 'webmock', '>= 1.18.0'
-  # for code quality checks
-  gem 'flog', '> 4.1.0'
-  # as driver for Capybara
-  # See test/test_helper.rb for details:
-  gem 'poltergeist', '>= 1.4'
-  # for mocking and stubbing
-  gem 'mocha', '> 0.13.0', require: false
-end
-
-# Gems used only during development not required in production environments by default.
-group :development do
-  # as alternative to the standard IRB shell
-  gem 'pry', '>= 0.9.12'
-  # to make rack(and rails) use unicorn by default
-  gem 'unicorn-rails' # webrick won't work
-  # for client side, DB and server profiling
-  gem 'rack-mini-profiler'
-  # for calling single testd
-  gem 'single_test'
-end
-
 # Gems used only for assets and not required in production environments by default.
 group :assets do
   # for minifying CSS
