Description: drop test and development depends.
Author: Andrew Lee (李健秋) <ajqlee@debian.org>

---
--- a/src/api/Gemfile
+++ b/src/api/Gemfile
@@ -66,65 +66,6 @@
 #  gem 'newrelic_rpm'
 end
 
-# Gems used only for testing the application and not required in production environments by default.
-group :test do
-  # as testing frameworks
-  gem 'minitest', '= 5.8.3'
-  gem 'test-unit'
-  # to ensure a clean state for testing
-  gem 'database_cleaner', '>= 1.0.1'
-  # for jenkins
-  gem 'ci_reporter'
-  # for test coverage reports
-  gem 'simplecov', require: false
-  gem 'coveralls', require: false
-  # for failing fast
-  gem 'minitest-fail-fast'
-  # for spec like reporting
-  gem 'minitest-reporters'
-  # for rspec like matchers
-  gem 'capybara_minitest_spec'
-  # to freeze time
-  gem 'timecop'
-  # to fake backend replies
-  gem 'webmock', '>= 1.18.0'
-  # for code quality checks
-  gem 'flog', '> 4.1.0'
-  # for mocking and stubbing
-  gem 'mocha', '> 0.13.0', require: false
-  # for testing common Rails functionality with simple one-liners
-  gem 'shoulda-matchers', '~> 3.1'
-end
-
-# Gems used only during development not required in production environments by default.
-group :development do
-  # as alternative to the standard IRB shell
-  gem 'unicorn-rails' # webrick won't work
-  # for calling single testd
-  gem 'single_test'
-  # as debugging tool in the default error page
-  gem 'web-console', '~> 2.0'
-end
-
-group :development, :test do
-  # as testing framework
-  gem 'rspec-rails', '~> 3.4.0'
-  # for fixtures
-  gem 'factory_girl_rails'
-  # for mocking the backend
-  gem 'vcr'
-  # as alternative to the standard IRB shell
-  gem 'pry', '>= 0.9.12'
-  # for style checks
-  gem 'rubocop', require: false
-  # to generate random long strings
-  gem 'faker'
-  # as driver for capybara
-  gem 'poltergeist', '>= 1.4'
-  # to launch browser in test
-  gem 'launchy'
-end
-
 # Gems used only for assets and not required in production environments by default.
 group :assets do
   # for minifying CSS
