--- a/src/api/Gemfile
+++ b/src/api/Gemfile
@@ -39,10 +39,6 @@
 gem 'responders', '~> 2.0'
 # needed for travis-ci.org, must be global for scripts
 gem 'bundler'
-# for threaded comments
-gem 'acts_as_tree'
-# js plotting (OBS monitor)
-gem 'flot-rails'
 
 group :development, :production do
   # to have the delayed job daemon
@@ -85,7 +81,7 @@
   # assets for jQuery and jQuery-ujs
   gem 'jquery-rails'
   # assets for jQuery-ui
-  gem 'jquery-ui-rails', '~> 4.2.1' # version 5 needs henne's new webui
+  gem 'jquery-ui-rails', '>= 5.0.5' # version 5 needs henne's new webui
   # assets for the bootstrap front-end framework. Used by the bratwurst theme
   # gem 'bootstrap-sass-rails'
   # assets for font-awesome vector icons
