Description: Added missing jquery.ui.widget
Author: Bob Ham <bob.ham@collabora.com>
Origin: Debian
Bug-Debian: https://bugs.debian.org/839144
Bug-Debian: https://bugs.debian.org/839143
Reviewed-By: Andrew Lee (李健秋) <ajqlee@debian.org>
Last-Update: 2016-10-20

--- open-build-service-2.7.1.orig/src/api/app/assets/javascripts/webui/application.js
+++ open-build-service-2.7.1/src/api/app/assets/javascripts/webui/application.js
@@ -11,6 +11,7 @@
 // GO AFTER THE REQUIRES BELOW.
 //
 //= require jquery
+//= require jquery.ui.widget
 //= require jquery.ui.menu
 //= require jquery.ui.autocomplete
 //= require jquery.ui.tabs
