Index: open-build-service-2.6-collabora/src/api/Gemfile
===================================================================
--- open-build-service-2.6-collabora.orig/src/api/Gemfile
+++ open-build-service-2.6-collabora/src/api/Gemfile
@@ -46,38 +46,27 @@ gem 'escape_utils'
 # as authorization system
 gem "pundit"
 
-group :production do
-  # if you have an account, it can be configured by
-  # placing a config/newrelic.yml
-  # be aware about the non-OSS license
-#  gem 'newrelic_rpm'
-end
-
-# Gems used only for assets and not required in production environments by default.
-group :assets do
-  # for minifying CSS
-  gem 'cssmin', '>= 1.0.2'
-  # for minifying JavaScript
-  gem 'uglifier', '>= 1.2.2'
-  # to use sass in the asset pipeline
-  gem 'sass-rails', '~> 4.0.3'
-  # assets for jQuery DataTables
-  gem 'jquery-datatables-rails'
-  # assets for the text editor
-  gem 'codemirror-rails', '>= 4.2'
-  # assets for jQuery tokeninput
-  gem 'rails_tokeninput', '>= 1.6.1.rc1'
-  # to create our sprite images/stylesheets
-  gem 'sprite-factory', '>= 1.5.2'
-  # to read and write PNG images
-  gem 'chunky_png'
-  # assets for jQuery and jQuery-ujs
-  gem 'jquery-rails'
-  # assets for jQuery-ui
-  gem 'jquery-ui-rails'
-  # assets for the bootstrap front-end framework. Used by the bratwurst theme
-  #gem 'bootstrap-sass-rails'
-  # assets for font-awesome vector icons
-  gem "font-awesome-rails"
-end
-
+# for minifying CSS
+gem 'cssmin', '>= 1.0.2'
+# for minifying JavaScript
+gem 'uglifier', '>= 1.2.2'
+# to use sass in the asset pipeline
+gem 'sass-rails', '~> 4.0.3'
+# assets for jQuery DataTables
+gem 'jquery-datatables-rails'
+# assets for the text editor
+gem 'codemirror-rails', '>= 4.2'
+# assets for jQuery tokeninput
+gem 'rails_tokeninput', '>= 1.6.1.rc1'
+# to create our sprite images/stylesheets
+gem 'sprite-factory', '>= 1.5.2'
+# to read and write PNG images
+gem 'chunky_png'
+# assets for jQuery and jQuery-ujs
+gem 'jquery-rails'
+# assets for jQuery-ui
+gem 'jquery-ui-rails'
+# assets for the bootstrap front-end framework. Used by the bratwurst theme
+#gem 'bootstrap-sass-rails'
+# assets for font-awesome vector icons
+gem "font-awesome-rails"
Index: open-build-service-2.6-collabora/src/api/config/application.rb
===================================================================
--- open-build-service-2.6-collabora.orig/src/api/config/application.rb
+++ open-build-service-2.6-collabora/src/api/config/application.rb
@@ -2,8 +2,7 @@ require File.expand_path('../boot', __FI
 
 require 'rails/all'
 
-# Assets should be precompiled for production (so we don't need the gems loaded then)
-Bundler.require(*Rails.groups(assets: %w(development test)))
+Bundler.require(*Rails.groups)
 require_relative '../lib/engines/base.rb'
 OBSEngine::load_engines
 
