diff -ruN buildservice-1.0.0-orig/src/sign/sign.conf buildservice-1.0.0/src/sign/sign.conf
--- buildservice-1.0.0-orig/src/sign/sign.conf	2008-07-07 06:02:00.000000000 +0000
+++ buildservice-1.0.0/src/sign/sign.conf	2008-07-10 11:56:56.111227934 +0000
@@ -1,3 +1,8 @@
-allow: (incoming IPs, one per line)
-user: build@suse.de
-server: (signing server)
+### for build service host
+#server: <IP of your signd host>
+#user: build@suse.de
+#allowuser: obsrun
+#
+### for sign server
+#allow: <IP of your build service host>
+#phrases: /root/.phrases
