diff --git a/src/api/app/views/webui/main/index.html.haml b/src/api/app/views/webui/main/index.html.haml
index 48b4e04..38deb1f 100644
--- a/src/api/app/views/webui/main/index.html.haml
+++ b/src/api/app/views/webui/main/index.html.haml
@@ -18,7 +18,6 @@
         = proceed_link "document-new", 'New Project', controller: 'project', action: 'new', ns: User.current.home_project_name unless User.current.is_nobody?
       - unless @spider_bot
         = proceed_link 'utilities-system-monitor', 'Status Monitor', controller: 'monitor', action: 'index'
-      = proceed_link 'drive-optical-32', 'New Image', image_templates_path if Feature.active?(:image_templates)
   = render partial: 'systemstatus'
 .grid_5.omega
   - if User.current.is_nobody? and can_register
